packages/wrangler/bin/wrangler.js

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env node
 const { spawn } = require("child_process");
-const { join } = require("path");
+const path = require("path");
+const fs = require("fs");
+const os = require("os");
```

Comment:
Should we be using the `node:` prefixed paths for these?

Reply:
that'll fail in older versions of node, and we eon't be able to show the message that asks to update their version of node (see https://github.com/cloudflare/wrangler2/pull/980)